onnxruntime/core/optimizer/conv_activation_fusion.h

```diff
@@ -9,10 +9,11 @@ namespace onnxruntime {
 
 class ConvActivationFusion : public onnxruntime::GraphTransformer {
  public:
-  ConvActivationFusion() noexcept : onnxruntime::GraphTransformer("ConvActivationFusion", "Fusing Activation into Conv") {}
+  ConvActivationFusion() noexcept : onnxruntime::GraphTransformer("ConvActivationFusion", "Fusing Activation into Conv", TransformerLevel::Optional_L2, 
+      std::vector<std::string>{onnxruntime::kCpuExecutionProvider}) {}
```

Comment:
>std::vector<std::string> [](start = 6, length = 24)

FWIW I don't think you need to explicitly put the 'std::vector<std::string>' here. Up to you if you want to.